index.html

```diff
@@ -21,18 +21,22 @@
     <![endif]-->
   </head>
   <body ng-app="GiphyApp">
-    <div class="container">
+    <div class="container" ng-controller="SearchCtrl">
       <h1 class="text-center">Welcome to Giphy Search</h1>
 
       <form class="form-inline">
         <div class="form-group">
-          <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+          <input type="text" ng-model="searchTerm" placeholder="Search for GIFs" id="search" class="form-control">
+          <button type="submit" ng-click="search()" class="btn btn-primary">Search</button>
         </div>
-        <button type="submit" class="btn btn-primary">Search</button>
       </form>
 
-      <div id="results"></div>
+      <div id="results" ng-repeat="gif in gifs">
+        <!-- <a href="{{gif.url}}" -->
```

Comment:
Missing a ```>``` on the end of your anchor tag, putting it in should fix the link problem!